### PR TITLE
Update for 64-bit

### PR DESCRIPTION
**Pull request by jamesche75**
Added info on how you can start a 64-bit app using the 64-bit version of dotnet.exe.

### CODE REVIEW

**Comment by guardrex** [comment]
Thanks @jamesche75 ... I'll take a look at this over the weekend.

**Comment by guardrex** [comment]
@jamesche75 Can you sign the CLA :point_up:.

[@Rick-Anderson @scottaddie @tdykstra ... another example of the CLA notice appearing late. I made a comment and an update, then the CLA notice appeared.]

**Comment by Rick-Anderson** [comment]
@jamesche75 Can you sign the CLA?

**Comment by jamesche75** [comment]
> @jamesche75 Can you sign the CLA?

Done.

File: aspnetcore/host-and-deploy/azure-apps/troubleshoot.md [hunk 1]
```diff
@@ -65,12 +65,23 @@ Many startup errors don't produce useful information in the Application Event Lo
...
-   * If the app is a [self-contained deployment](/dotnet/core/deploying/#self-contained-deployments-scd), run the app's executable. In the following command, substitute the name of the app's assembly for `<assembly_name>`: `<assembly_name>.exe`
+   * If the app is a [framework-dependent deployment](/dotnet/core/deploying/#framework-dependent-deployments-fdd), run the app's assembly with `dotnet .\{ASSEMBLY NAME}.dll`.
+   * If the app is a [self-contained deployment](/dotnet/core/deploying/#self-contained-deployments-scd), run the app's executable with `{ASSEMBLY NAME}.exe`.
 1. The console output from the app, showing any errors, is piped to the Kudu console.
 
+#### Test a 64-bit (x64) app
+
+* If the app is a 64-bit (x64) [framework-dependent deployment](/dotnet/core/deploying/#framework-dependent-deployments-fdd), run the app using the 64-bit *dotnet.exe* located in the `D:\Program Files\dotnet` directory:
```
Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/azure-apps/troubleshoot.md]: Why is this so verbose?

If the app is a 64-bit (x64) [framework-dependent deployment](/dotnet/core/deploying/#framework-dependent-deployments-fdd):

1. cd *D:\Program Files\dotnet*
1. `dotnet \home\site\wwwroot\{ASSEMBLY NAME}.dll`.
 * The console output from the app, showing any errors, is piped to the Kudu console.
 * If the app is a [self-contained deployment](/dotnet/core/deploying/#self-contained-deployments-scd), run the app's executable with `{ASSEMBLY NAME}.exe`.

Comment (guardrex) [reply aspnetcore/host-and-deploy/azure-apps/troubleshoot.md]: I wouldn't characterize it that way ... I'd just call it *conversational tone*. However, one author's "conversational tone" is another author's "verbose language." :smile: lol

I'll revise it. Stand-by.

Comment (jamesche75) [reply aspnetcore/host-and-deploy/azure-apps/troubleshoot.md]: It explains why you need to change into the directory specified. As a consumer of docs, I prefer you to tell me why I'm doing something rather than just telling me with no explanation. Up to you.

Comment (guardrex) [reply aspnetcore/host-and-deploy/azure-apps/troubleshoot.md]: It doesn't present clear instructions to have "cd D: ..." ... readers (e.g., newbies) could just totally choke on that.

Comment (guardrex) [reply aspnetcore/host-and-deploy/azure-apps/troubleshoot.md]: @jamesche75 I lean that way in just about everything I write, but I know that sometimes I get too wordy. There are some cuts that can be made here. I'll have another commit up in a few minutes, and we can take a look at a shorter version and revise further if needed.

**Comment by guardrex** [comment]
@Rick-Anderson See :eyes: now ... shorter than before.

**Comment by guardrex** [comment]
@glennc Can you remark on this process for an app that's on the preview site extension. It doesn't work for that scenario using these steps. Wondering if it's even possible? If so, how?

**Comment by glennc** [comment]
The preview runtime is not installed in Program Files. So you would need to change the step that paths to the dotnet.exe to use am x64 one that comes with the site extension. But I am not sure that we actually carry both in the site extension.

**Comment by guardrex** [comment]
mmmmm 🤔 .... it's a edge case .... but perhaps an important one given that if something is going to break or surprise a dev it may very likely happen when testing new preview bits.

>  one that comes with the site extension

I looked around a little bit for it but didn't see it. Do you know the path to it? ... or who would know? If I can at least get the path, then we have a shot of covering it ... if you want to cover it ... your call.

**Comment by guardrex** [comment]
*Geebers McScreebers!*:tm: ... I didn't look too hard last time. *It was staring me in the face!*

capture

**Comment by guardrex** [comment]
@glennc Is this correct (last commit)? ...

https://github.com/aspnet/Docs/pull/11230/commits/822d589b6443d5439d2085114321bafadfd66371

I'm only going off of the presence of this folder/*dotnet.exe* ... I haven't manually tested with a 3.0 preview app, but I can do that if you have any concerns about this working.

**Comment by guardrex** [comment]
@Rick-Anderson I just tested the **_New!_** steps for a preview runtime. :tada:

It's ready to merge unless u have changes.

File: aspnetcore/host-and-deploy/azure-apps/troubleshoot.md [hunk 2]
```diff
@@ -65,11 +65,48 @@ Many startup errors don't produce useful information in the Application Event Lo
...
+1. Run the app: `dotnet \home\site\wwwroot\{ASSEMBLY NAME}.dll`.
+
+The console output from the app, showing any errors, is piped to the Kudu console.
+
+#### Test a 64-bit (x64) app
+
+##### Current release
+
+* If the app is a 64-bit (x64) [framework-dependent deployment](/dotnet/core/deploying/#framework-dependent-deployments-fdd):
+  1. Execute `cd D:\Program Files\dotnet`.
```
Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/azure-apps/troubleshoot.md]: 1. `cd D:\Program Files\dotnet`.

Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/azure-apps/troubleshoot.md]: This is an advanced tutorial. Execute is not approved on pollicheck. I don't think it MT well. It adds nothing and subtracts much.

cc @tdykstra

File: aspnetcore/host-and-deploy/azure-apps/troubleshoot.md [hunk 3]
```diff
@@ -65,11 +65,48 @@ Many startup errors don't produce useful information in the Application Event Lo
...
+  1. Run the app with `dotnet \home\site\wwwroot\{ASSEMBLY NAME}.dll`.
```
Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/azure-apps/troubleshoot.md]: Run the app: `dotnet \home\site\wwwroot\{ASSEMBLY NAME}.dll`

File: aspnetcore/host-and-deploy/azure-apps/troubleshoot.md [hunk 4]
```diff
@@ -65,11 +65,48 @@ Many startup errors don't produce useful information in the Application Event Lo
...
+* If the app is a [self-contained deployment](/dotnet/core/deploying/#self-contained-deployments-scd):
+  1. Execute `cd D:\home\site\wwwroot`.
+  1. Run the app with `{ASSEMBLY NAME}.exe`.
```
Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/azure-apps/troubleshoot.md]: remove period on all these.
~with~
